docs/source/methods/variance_reduction.rst

```diff
@@ -0,0 +1,129 @@
+.. _methods_variance_reduction:
+
+==================
+Variance Reduction
+==================
+
+.. _methods_variance_reduction_intro:
+
+------------
+Introduction
+------------
+
+Transport problems can sometimes infolve a significant degree of attenuation
+between the neutron source and a detector (tally) region, which can result in a
+flux differential of ten orders of magnitude (or more) throughout the simulation
+domain. As Monte Carlo uncertainties tend to be inversely proportional to the
+physical flux density, it can be extremely difficult to accurately resolve
+tallies in locations that are optically far from the source. This issue is
+particularly common in fixed source simulations, where some tally locations may
+not experience a single scoring event, even after billions of analog histories.
+
+Variance reduction techniques aim to either flatten the global uncertainty
+distribution, such that all regions of phase space have a fairly similar
+uncertainty, or to reduce the uncertainty in specific locations (such as a
+detector). There are two strategies available in OpenMC for variance reduction:
+the Monte Carlo MAGIC method, and with FW-CADIS. Both strategies work by
+developing a weight window mesh, which can be utilized by subsequent Monte Carlo
+solves to split particles heading towards areas of lower flux densities while
+terminating particles in higher flux regions -- all while maintaining a fair
+game.
+
+------------
+MAGIC Method
+------------
+
+The MAGIC method is an iterative technique that uses spatial flux information
+(:math:`\phi(r)`) obtained from a normal Monte Carlo solve to produce weight
+windows (:math:`\w(r)`) that can be utilized by a subsequent iteration of Monte
```

Comment:
This math doesn't seem to be rendering properly in the RTD build.

Reply:
Thanks! Yes, there was an errant `\` before the w that has now been removed.